### PR TITLE
update to new push notification registration

### PR DESCRIPTION
**Pull request by tristan**
fixes up breaking changes that will be pushed to dev soon: https://github.com/tokenbrowser/token-ethereum-service/pull/5

not tested cause i can't xcode! i'll not merge my backend code until it's verified i've not messed something up here and someone can test this is fine 🙈

### CODE REVIEW

File: Token/APIClients/EthereumAPIClient.swift [hunk 1]
```diff
@@ -172,8 +172,9 @@ public class EthereumAPIClient: NSObject {
             let cereal = Cereal.shared
             let path = "/v1/apn/register"
             let address = cereal.paymentAddress
+            let paymentAddress = TokenUser.current!.paymentAddress
```
Comment (Elland) [inline Token/APIClients/EthereumAPIClient.swift]: @tristan `TokenUser.current!.paymentAddress` and `cereal.paymentAddress` are the same. 😅 But no worries. Code is not super clear here.

Comment (tristan) [inline Token/APIClients/EthereumAPIClient.swift]: haha, that's what i was trying to ask you about yesterday!

Comment (Elland) [inline Token/APIClients/EthereumAPIClient.swift]: I should've just walked over to your desk and checked. 😅 That's what we get, working after travelling for 7,5 hours, hehe.

**Comment by Elland** [comment]
@tristan made a very small change, it should all work now. Can you make sure I got that right, and that token id for the http header should be the ID address, not the payment address?

**Comment by Elland** [comment]
Perfect. Merging! 🚢

**Pull request merged**